PerimeterXModule/Internals/Helpers/RemoteServer.cs

```diff
@@ -0,0 +1,194 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.Specialized;
+using System.Configuration;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Net;
+using System.Text;
+using System.Threading.Tasks;
+using System.Web;
+
+namespace PerimeterX
+{
+	internal class RemoteServer
+	{
+		string _remoteUrl;
+		HttpContext _context;
+
+		/// <summary>
+		/// Initialize the communication with the Remote Server
+		/// </summary>
+		/// <param name="context">Context</param>
+		public RemoteServer(HttpContext context, string serverUrl, string uri)
+		{
+			_context = context;
+			_remoteUrl = serverUrl + uri;
```

Comment:
After all these `string.Format` eventually here it's a `+`? 😂

Reply:
trying to avoid string formats